kraken/chaos_recommender/prometheus.py

```diff
@@ -79,7 +70,7 @@ def fetch_utilization_from_prometheus(prometheus_endpoint, auth_token, namespace
     cpu_limits_result = prometheus.custom_query(cpu_limits_query)
 
 
-    mem_query = 'sum by (pod) (avg_over_time(container_memory_usage_bytes{image!="", namespace="%s"}[%s]))' % (namespace, duration)
+    mem_query = 'sum by (pod) (avg_over_time(container_memory_usage_bytes{image!="", namespace="%s"}[%s]))' % (namespace, scrape_duration)
```

Comment:
@mudverma this method was taking the scrape duration as parameter but was using a constant defined on top of the flle, I've restored the parameter passed by the CLI/config that otherwise would be unused.